webcompat/static/js/lib/bugform.js

```diff
@@ -558,6 +560,7 @@ function BugForm() {
             "The image is too big! Please choose something smaller than 4MB.";
           wcEvents.trigger("flash:error", { message: msg, timeout: 5000 });
         }
+        this.loadingIndicator.removeClass("is-active");
```

Comment:
I would stick this under `this.uploadLoader.removeClass(".is-active");`, but let's file a follow up "good first bug".